DSD100_gather_hpc.m

```diff
@@ -0,0 +1,29 @@
+base_estimates_directory = '/home/marius/Documents/Database/DSD100/outputs/measures/'; 
```

Comment:
I think it would be nicer to have a shorter path here like ```'measures/'``` (also removing your local folder structure) ;-)